eva-lib/src/main/java/uk/ac/ebi/eva/lib/MongoConfiguration.java

```diff
@@ -29,12 +29,14 @@
 import org.springframework.data.mongodb.core.mapping.MongoMappingContext;
 
 import uk.ac.ebi.eva.commons.models.converters.data.DBObjectToVariantEntityConverter;
+import uk.ac.ebi.eva.lib.config.EvaProperty;
 
 import java.io.IOException;
 import java.util.ArrayList;
 import java.util.List;
 
 @Configuration
+@Import(EvaProperty.class)
```

Comment:
`DBAdaptorConnector` doesn't need to import this class to inject it, please check when is it really necessary.